### PR TITLE
Prepare 2.1.0

### PR DESCRIPTION
**Pull request by rcernich**
Updates version to 2.1.0
Initial generation of charts for v2.1
Updates v2.0 to 2.0.1.1

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -218,19 +247,19 @@ ifneq "${OSSM_MANIFEST_DATE}" ""
 endif
 
 .PHONY: update-charts
-update-charts: update-1.0-charts update-1.1-charts update-2.0-charts
+update-charts: update-1.0-charts update-1.1-charts update-2.0-charts update-2.1-charts
```
Comment (jwendell) [inline Makefile:250]: Can we drop 1.0 yet?

Comment (rcernich) [reply Makefile:250]: We should do that as part of a separate pr/commit.

**Review by dgn — APPROVED** [review]

**Pull request merged**